lib/topology/platforms/shell.py

```diff
@@ -676,7 +710,7 @@ class PExpectBashShell(PExpectShell):
 
     def __init__(
             self,
-            initial_prompt='\w+@.+:.+[#$] ', try_filter_echo=False,
+            initial_prompt='\w+@.+[#$] ', try_filter_echo=False,
```

Comment:
Please handle your specific prompts in a specific node, changing the default prompt of this class can affect other nodes that are already depending on it.